spec/models/subject_set_spec.rb

```diff
@@ -70,7 +70,7 @@
       end
     end
 
-    describe "#set_member_subjects_count" do
+    describe "#set_member_subjects_count", :disabled do
```

Comment:
Yes. The counter won't run right away (or at all since its in a test) so this won't be consistent immediately.